CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+2. It works.
+3. It follows the code style for [C/C++](http://dev.minetest.net/Code_style_guidelines) or [Lua](http://dev.minetest.net/Lua_code_style_guidelines).
+4. The code's internal interfaces are reasonably optimized, depending on how often the code is called.
+5. It uses protocols and formats which include the required compatibility.
+
+## Issues
+
+If you experience an issue, we would like to know the details - especially when a stable release is on the way.
+
+1. Do a quick search on GitHub to check if the issue has already been reported.
+2. Is it an issue with the Minetest *engine*? If not, report it [elsewhere](http://www.minetest.net/development/#reporting-issues).
+3. [Open an issue](https://github.com/minetest/minetest/issues/new) and describe the issue you are having - you could include:
+ - Error logs (check the bottom of the `debug.txt` file)
+ - Screenshots
+ - Ways you have tried to solve the issue, and whether they worked or not
+ - Your Minetest version and the content (subgames, mods or texture packs) you have installed
+ - Your platform (e.g. Windows 10 or Ubuntu 15.04 x64)
+
+After reporting you should aim to answer questions or clarifications as this helps pinpoint the cause of the issue (if you don't do this your issue may be closed after 1 month).
+
+## Feature requests
+
+Feature requests are welcome but take a moment to see if your idea follows the [roadmap](https://forum.minetest.net/viewtopic.php?t=9177) in some way and fits the whole picture of the project. You should provide a clear explanation with as much detail as possible.
+
+## Translations
+
+Translations of Minetest are performed using Weblate. You can access the project page [here](https://hosted.weblate.org/projects/minetest/minetest/).
+
+Current languages can be found in the `po/` directory, create an issue if you'd like to translate into a language that isn't there.
+
+### Donations
+
+If you'd like to monetarily support Minetest development, you can find donation methods on [our website](http://www.minetest.net/development/#donate).
+
+# Maintaining
+
+*This is a concise version of the [Rules & Guidelines](http://dev.minetest.net/Category:Rules_and_Guidelines) on the developer wiki.*
+
+These notes are for those who have push access Minetest (core developers / maintainers).
+
+- See the [project organisation](http://dev.minetest.net/Organisation) for the people involved.
+
+## Reviewing pull requests
+
+Pull requests should be reviewed and, if appropriate, checked if they achieve their intended purpose. You can show that you are in the process of, or will review the pull request by commenting *"Looks good"* or something similar.
+
+**If the pull-request is not [merge-able](#a-pull-request-is-considered-merge-able-when):**
+
+Submit a comment explaining to the author what they need to change to make the pull-request merge-able.
+
+- If the author comments or makes changes to the pull-request, it should be reviewed again.
```

Comment:
I changed it to use the word "can" so it should be less strict https://github.com/minetest/minetest/pull/3143/commits/87b9abc02ca9ea76daf131eab1f3ae5e2b2ec530